pom.xml

```diff
@@ -50,9 +50,9 @@
 			<scope>test</scope>
 		</dependency>
 		<dependency>
-			<groupId>org.simpleframework</groupId>
+			<groupId>com.subshell.simpleframework</groupId>
```

Comment:
@lags What does this fork of simple-xml do? Fix https://github.com/openpnp/openpnp/issues/1363 perhaps? I looked through https://github.com/subshell/simplexml but didn't see anything.

Reply:
Hi @vonnieda yes, this fork solve the errors raise when try to use Java 17. At the time I too hadn't found anything related [here](https://github.com/ngallagher/simplexml/compare/master...subshell:simplexml:master) but it works.
Probably the version upgrade has dependencies that fix the compatibility?